packages/blade/src/components/Amount/_decisions/decisions.md

```diff
@@ -0,0 +1,67 @@
+# Amount <!-- omit in toc -->
+
+Amount component is a UI element that displays and formats various currency values in a specified format.
+
+<img  src="./amount-thumbnail.png"  width="380" />
+
+## Design
+
+- [Figma - Amount](https://www.figma.com/file/jubmQL9Z8V7881ayUD95ps/Blade---Payment-Light?node-id=28012%3A580578&t=3peAz8A2n2Gw4WMl-1)
+
+## API
+
+| Prop           | Type                                                               | Default     | Description                                                                    | Required |
+| -------------- | ------------------------------------------------------------------ | ----------- | ------------------------------------------------------------------------------ | -------- |
+| children       | `string`                                                           | `undefined` | The value to be rendered within the component                                  | ✅       |
+| intent         | `default`, `positive`, `negative`, `notice`, `information`, `blue` | `default`   | The variant of the amount to be rendered                                       |          |
+| contrast       | `low`, `high`                                                      | `low`       | The contrast of the amount to be rendered                                      |          |
+| fontWeight     | `bold`, `regular`                                                  | `bold`      | The weight of the amount to be rendered                                        |          |
+| hasDecimals    | `true`, `false`                                                    | `true`      | Indicates whether the amount has or can have decimals                          |          |
+| hasTextPostfix | `true`, `false`                                                    | `true`      | Indicates whether a text postfix should be used                                |          |
```

Comment:
text postfix seems like a very vague terminology. As a dev I'm not able to visualize what this would do? I think we should use a clearer terminology. Same can be communicated to design and updated in figma.

Reply:
changed it to **showCurrencyPostfix**

Reply:
Sounds good, can you please communicate and get that updated on figma too?